### PR TITLE
ALC-197 - add weight information to header

### PR DESCRIPTION
**Pull request by hosainnet**
Fetch initial and gradual weight information from etherscan

- [x] Branch name corresponds to Linear ticket number.
- [x] Added sufficient integration and unit tests.
- [x] Addressed a single concern in the least number of changed lines as
      possible.
- [x] Only fixed/added the functionality in question **OR** address wide-spread
      whitespace/style issues, not both.

### CODE REVIEW

File: .env.local.example [hunk 1]
```diff
@@ -1 +1,2 @@
-REACT_APP_INFURA_API_KEY=xxxs
\ No newline at end of file
+REACT_APP_INFURA_API_KEY=xxx
+REACT_APP_ETHERSCAN_API_KEY=xxx
```
Comment (ledes) [inline .env.local.example:2]: do you need a key? 
If we don't provide one, it uses a free one by default. We currently have that for poke weights

File: src/components/PoolHeader/index.tsx [hunk 2]
```diff
@@ -21,10 +24,24 @@ export default function Index({ poolDetail }: PoolHeaderProps) {
...
+  useEffect(() => {
+    getPoolWeights(poolDetail.crpControllerAddress, poolDetail.totalWeight)
+      .then((poolWeights: PoolWeights) => {
+        setStartWeights(poolWeights.startWeights);
+        setEndWeights(poolWeights.endWeights);
+      })
+      .catch((e: Error) => {
+        console.log(e.message);
+      });
+  }, []);
```
Comment (ledes) [inline src/components/PoolHeader/index.tsx:39]: does it need to depend on poolDetail

```
...

}, [poolDetail]);
```

File: .env.local.example (hunk 1 above)
Comment (hosainnet) [reply .env.local.example:2]: Yeah it'll use the free one if you don't set up this env var

Comment (ledes) [reply .env.local.example:2]: is the one you use here a paid one?, otherwise you don't need this env variable

Comment (hosainnet) [reply .env.local.example:2]: I found sometimes that I get throttled locally if I make too many calls, so I put my own free key which I think gives us better allowance? I can maybe put a comment next to this line to say that this is optional

Comment (ledes) [reply .env.local.example]: ohh ok... then I guess I should add it to the poke weights one too.
We can do that once we put them in the chart

**Review by ledes — APPROVED** [review]

File: .env.local.example (hunk 1 above)
Comment (hosainnet) [reply .env.local.example]: If we use the same etherscan service / provider, it'll apply everywhere ;)

Comment (hosainnet) [reply .env.local.example]: https://github.com/alchemistcoin/copper/pull/18/files#diff-bef7c606da79173e54bf27f331eeaebfd827bcbfd1d4181186d2507c2c919666R29-R32

**Pull request merged**